setup.py

```diff
@@ -153,6 +153,8 @@ def run(self):
         'python-dateutil',
     ],
     extras_require={
-        'mitutoyo': []
```

Comment:
Hypothetically, you want to keep those empty extras once they had been introduced.

Reply:
What do you need them for?

Reply:
Ah, once they had been introduced in a prior version such that install SurfaceTopography [mitutoyo] continues to work? Why is the openpyxl dependency not in there?